### ISSUE TITLE
Extract MLST sequences for trees

### ISSUE THREAD

**Issue by TseWue** [tseemann/mlst#93]
Hi Thorsten

Thank you so much for this handy tool. I was wondering if there is a possibility to also save the (concatenated) sequences of the identified MLST genes to file? Similarly to the novel alleles but for all of the identified gene sequences at once? It would be very handy to create MLST trees.

Best wishes,
Tse

**Comment by tseemann** [comment]
Best to draw the tree using the MLST allele numbers.
With `--legacy --scheme xxx` output it should be easy to put into PhyloViz.

**Comment by TseWue** [comment]
Yes I thought this would be the thing to do but since some matches sometimes are labelled with ~ or ? it could lead to minor falsehoods in the tree. But they won't have much of an influence I guess, the type is still the type. Thanks and stay safe!
Tse

**Comment by tseemann** [comment]
Yes I see your point.

I am planning to assign new "IDs" for novel alleles, and use them for all matches in a set if you run `mlst *.fasta` etc.

**Comment by avonm** [comment]
Hi, 
See the ticket is still open. Any chance this issue is already handled?
Thanks!
//Astrid

**Comment by nweerasu** [comment]
Following for the same reason -- ty
Nimalka
